frontend/src/pages/_events/slug/register/index.js

```diff
@@ -343,6 +347,10 @@ export default RequiresPermission(() => {
                             <RegistrationSection
                                 isActive={activeStep === index}
                                 data={formData}
+                                sectionsInfo={{
+                                    sections: sections.length,
+                                    index: index + 1,
+                                }}    
```

Comment:
*Optional*
Since the 'index' property, is going to refer to the active section, you should change the property name to something more descriptive like for example `activeSection`